packages/opentelemetry-core/src/trace/sampler/ProbabilitySampler.ts

```diff
@@ -18,15 +18,21 @@ import { Sampler, SpanContext } from '@opentelemetry/types';
 
 /** Sampler that samples a given fraction of traces. */
 export class ProbabilitySampler implements Sampler {
-  readonly description = 'ProbabilitySampler';
-
   constructor(private readonly _probability: number = 1) {}
 
   shouldSample(parentContext?: SpanContext) {
     if (this._probability >= 1.0) return true;
     else if (this._probability <= 0) return false;
     return Math.random() < this._probability;
   }
+
+  toString(): string {
+    // TODO: Consider to use `AlwaysSampleSampler` and `NeverSampleSampler`
+    // based on the specs.
+    return `ProbabilitySampler{${
+      this._probability > 1 ? 1 : this._probability
```

Comment:
This doesn't handle values less than `0`. Maybe it would be clearer to use `Math.floor(Math.ceil())`?

Reply:
I think we want to preserve the probability rate (ex: 0.0010 or 0.1), The `Math.ceil()` function always rounds a number up to the next largest integer. With this, above rate will become 1, which doesn't make sense to me. WDYT?

Reply:
Could we make it normalize `_probability` in the constructor and then just assume it's between 0 and 1 in `toString` and in the sample function?

Reply:
> With this, above rate will become 1, which doesn't make sense to me.

Correct, I forgot it doesn't handle floats. In any case, the main concern is that values less than `0` should become `0`. I agree with @draffensperger that this should only be done once.

Reply:
Done in c95fa388c28b0e342bab3768f482f6f83d4cdc1e